config/environments/staging.rb

```diff
@@ -39,7 +39,12 @@
   # `config.assets.precompile` and `config.assets.version` have moved to config/initializers/assets.rb
 
   # Enable serving of images, stylesheets, and JavaScripts from an asset server.
-  # config.action_controller.asset_host = "http://assets.example.com"
+  if ENV["PUBLIC_URL"].present?
+    # example: https://publishers-staging-distro.basicattentiontoken.org
+    config.public_file_server.headers = {
+      'Cache-Control' => 'public, max-age=31536000'
```

Comment:
I'm not certain this is the right configuration. My impression is that we should setup a CDN to speedup the load for assets. Would this affect all assets we upload inclusive of javascript?

Reply:
My original config was to actually serve all assets from publishers-distro with the following
```rb
config.action_controller.asset_host = ENV["PUBLIC_URL"]
```

However that caused issues when loading fonts. I don't think this cache control adds that much for our speed, it was just one of the recommendations that the PageSpeed utility told us that might have a benefit and seemed to increase the score a bit. 

This would impact assets we upload along with javascript. I don't think this configuration has very much impact though as the filenames receive a unique hash that vary from deploy to deploy.

Reply:
I'm just reverting this back to staging, no reason to keep this being a problem